ci-operator/step-registry/ipi/aws/pre/sharednetwork/OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
```

Comment:
should probably include @jstuever and @patrickdillon based on [installer AWS approvers][1].  And possibly some folks in your current list aren't interested in being approvers for this specific directory (folks with a general interest in multi-step are probably already approvers from root-wards directories and don't need to be repeated down in this one).

[1]: https://github.com/openshift/installer/blob/145e5ccdf32513ee96c777853e1a7a7620781960/OWNERS_ALIASES#L52-L53